src/landscape/components/LandscapeForm/ProfileStep.js

```diff
@@ -66,10 +68,8 @@ const AreaTypesHelperText = props => {
 
 const EcosystemTypesHelperText = () => {
   return (
-    <Box sx={{ p: 2 }}>
-      <Typography>
-        <Trans i18nKey="landscape.form_profile_ecosystem_types_helper_text" />
-      </Typography>
+    <Box sx={{ p: 2, fontFamily: theme.typography.fontFamily }}>
```

Comment:
Suggestion: You don't need to import the theme you have it in side the sx property
```suggestion
    <Box sx={theme => ({ p: 2, fontFamily: theme.typography.fontFamily })}>
```